client/src/app/+video-channels/video-channel-edit/video-channel-update.component.ts

```diff
@@ -95,7 +95,7 @@ export class MyVideoChannelUpdateComponent extends MyVideoChannelEdit implements
 
           this.notifier.success($localize`Video channel ${videoChannelUpdate.displayName} updated.`)
 
-          this.router.navigate([ '/my-library', 'video-channels' ])
+          this.router.navigate([ '/c', this.videoChannel.name ])
```

Comment:
Prefer to redirect to previous route (`redirect.service.ts`)

If there is no previous route, you could redirect to `this.router.navigate([ '/c', this.videoChannel.name ])`